naplib/features/peak_rate.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from scipy.signal import butter, filtfilt, find_peaks
+from scipy.stats import zscore
+
+
+def peak_rate(aud, sfreq, band=[1,10], thresh=0):
+    '''
+    Extract peak_rate events from the (auditory) spectrogram
```

Comment:
Just so it links to the reference
```suggestion
    Extract peak_rate events [#1oganian]_ from the (auditory) spectrogram
```